pydantic/networks.py

```diff
@@ -86,136 +88,294 @@ def __hash__(self) -> int:
             )
         )
 
+    @property
+    def set_constraints(self) -> dict[str, Any]:
```

Comment:
```suggestion
    def defined_constraints(self) -> dict[str, Any]:
```

maybe? set_constraints can be confusing at first, feels like this is representing an action of _setting_ something

Reply:
Good call